inc/class-wpseo-image-utils.php

```diff
@@ -381,16 +381,27 @@ public static function get_first_usable_content_image_for_post( $post_id = null
 		$image_finder = new WPSEO_Content_Images();
 		$images       = $image_finder->get_images( $post->ID, $post );
 
-		if ( ! is_array( $images ) || empty( $images ) ) {
-			return null;
-		}
+		return self::get_first_image( $images );
+	}
 
-		$image_url = reset( $images );
-		if ( ! $image_url ) {
+	/**
+	 * Gets the term's first usable content image. Null if none is available.
+	 *
+	 * @param int $term_id The term id.
+	 *
+	 * @return string|null The image URL.
+	 */
+	public static function get_first_content_image_for_term( $term_id = null ) {
+		if ( $term_id === null ) {
 			return null;
 		}
```

Comment:
Defaulting `$term_id` to `null` and then check for `null`  to return `null`  seems a bit redundant to me, unless I'm missing something. Wouldn't not defaulting to null and then the following be simpler?
```
	if ( ! $term_id ) {
		return null;
	}
```

Reply:
@andizer and I decided to remove this logic. The motivation for this removal is that the `term_description` function, which directly follows this logic, already returns an empty string in case of an error